tests/pyincore/analyses/wfnfunctionality/test_wfnfunctionality.py

```diff
@@ -0,0 +1,127 @@
+# This pre accompanyogram and thing materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+from pyincore import IncoreClient, FragilityService, MappingSet, NetworkDataset, DataService, Dataset
+from pyincore.analyses.montecarlofailureprobability import MonteCarloFailureProbability
+from pyincore.analyses.wfnfunctionality import WfnFunctionality
+from pyincore.analyses.pipelinedamagerepairrate import PipelineDamageRepairRate
+from pyincore.analyses.pipelinefunctionality import PipelineFunctionality
+from pyincore.analyses.waterfacilitydamage import WaterFacilityDamage
+
+
+def run_with_base_class():
+    client = IncoreClient()
+    data_service = DataService(client)
+
+    # Top-level hazard type
+    hazard_type = "earthquake"
+    hazard_id = "5b902cb273c3371e1236b36b"
+
+    # Water facility damage configuration
+    wfn_dataset_id = "62d586120b99e237881b0519"  # MMSA wft network
+    wf_dataset_id = "62cdd5371cca614f5242e635"
```

Comment:
These should now be changed to:
wfn_dataset_id = "62d584e8861e370172cd2eca"
wf_dataset_id = 62d85363861e370172d11ed0

Reply:
Done